tilings/strategies/fusion.py

```diff
@@ -443,6 +443,8 @@ def _update_subparams(
         """
         res = {self.fuse_parameter: number_of_left_points + number_of_right_points}
         for parameter, value in parameters.items():
+            if parameter not in self.extra_parameters and value != 0:
+                return None
```

Comment:
Everything will work fine if `parameter not in self.extra_parameters and value == 0`?

Reply:
This is because the assumption mapped to an empty assumption (such assumptions are not added to the extra_parameters). Therefore, it must be the case that this assumption has the value 0.